src/main/java/org/jenkinsci/plugins/workinghours/model/DateTimeUtility.java

```diff
@@ -70,6 +70,10 @@ public static LocalTime localTime(final String value) {
         return null;
     }
 
+    public static LocalTime localTime(final int minutes) {
```

Comment:
It might be good to rename this method for clarity, e.g.
```suggestion
    public static LocalTime localTimeFromMinutes(final int minutes) {
```